mender-update/inventory.cpp

```diff
@@ -82,16 +85,18 @@ error::Error PushInventoryData(
 	stringstream top_ss;
 	top_ss << "[";
 	auto inv_data = ex_inv_data.value();
-	for (const auto &kv : inv_data) {
+	auto key_vector = common::GetMapKeyVector(inv_data);
+	std::sort(key_vector.begin(), key_vector.end());
```

Comment:
Could also just switch over to `std::map`, which is sorted from the get-go, unlike `unordered_map`.

Reply:
Yes, but to me it makes more sense to keep the data unordered and then only sort it in the place where we need it sorted.